storybook/pages/ProfileShowcaseAccountsPanelPage.qml

```diff
@@ -21,91 +20,76 @@ SplitView {
 
     orientation: Qt.Vertical
 
-    Popups {
-        popupParent: root
-        rootStore: QtObject {}
-        communityTokensStore: CommunityTokensStore {}
-    }
-
     readonly property string currentWallet: "0xcdc2ea3b6ba8fed3a3402f8db8b2fab53e7b7420"
 
     ListModel {
-        id: emptyModel
-    }
-
-    ListModel {
-        id: accountsModel
-
+        id: hiddenModelItem
         ListElement {
             name: "My Status Account"
-            address: "0xcdc2ea3b6ba8fed3a3402f8db8b2fab53e7b7420"
+            key: "0xcdc2ea3b6ba8fed3a3402f8db8b2fab53e7b7420"
```

Comment:
that's true. But in the ProfileShowcasePanel we're expecting some common rolenames from all models. The minimum necessary to provide the common features (row move and visibility change). These role names will be provided by an UI component ([ui/app/AppLayouts/Profile/helpers/ProfileShowcaseModelAdapter.qml](https://github.com/status-im/status-desktop/pull/13769/files#diff-7122972f7574cde0f42032835a03ea6f7c3d1772fd49feab3f267c7dbb9f9a33R1)).

Reply:
In case this the joined model, i think it's better to use specific role name (`address `, `communityId` and etc)

Reply:
Makes sense 🤔 `key` could become a proxy role instead of renaming the original role. I like the idea, but not sure now it's worth adding another SFPM on top for this. I think it's a good approach to keep the number of proxies to a minimum. I'll look into this!

Reply:
Mmmm, yeah, I'm also thinking on it now.. if the cost of adding the SFPM is not a lot, maybe it's worth it to keep the original role bc, indeed, we will need this role for example to create the search filters criterias.. and it will me more descriptive. 

HOWEVER... I'm thinking that this `key` value should come from the nim backend @MishkaRogachev... In case of wallet accounts, the key is just the wallet address but in case of collectibles the key should be the combination (hash or something) between 3 values (contractAddress, tokenId and chainId). From mpov, this unique identifier should be provided by this layer and be the same accross UI and NIM.